test/java/org/opendatakit/briefcase/reused/http/CommonsHttpTest.java

```diff
@@ -156,4 +157,12 @@ public void can_handle_3xx_errors() throws Exception {
       assertThat(response.isSuccess(), is(false));
     });
   }
+
+  @Test
+  public void the_factories_accept_max_http_connections_with_Proxy(){
```

Comment:
I think the test method name has to be improved. This test verifies that the returned object is of a certain type, which doesn't have to do with the method name. It's very important that the test methods convey with precission what's being verified.